packages/react/src/modules/slack/components/ConnectedSlackChannelPicker/ConnectedSlackChannelPicker.tsx

```diff
@@ -111,39 +96,22 @@ export const ConnectedSlackChannelPicker: React.FC<Props> = ({
             key={"select"}
             className="rnf-header"
             onClick={() => {
-              if (!errorMessage) {
```

Comment:
will be introducing errors back when we have a context for the connected state of these components, doesn't make sense to deal with it in this PR